Dockerfile

```diff
@@ -34,7 +34,8 @@ WORKDIR /srv
 
 # Install python and import python dependencies
 RUN apt-get update && apt-get install --no-install-recommends --yes python3 python3-lib2to3 python3-pkg-resources
-COPY --from=python-dependencies /root/.local/lib/python3.7/site-packages /root/.local/lib/python3.7/site-packages
+COPY --from=python-dependencies /root/.local/lib/python3.*/site-packages /tmp/site-packages
+RUN mkdir -p `python3 -m site --user-site` && rmdir `python3 -m site --user-site` && mv /tmp/site-packages `python3 -m site --user-site`
```

Comment:
I think this would require a comment before. I looks like a magic spell :stuck_out_tongue:

Reply:
@tbille made it slightly less magic, and gave it far too much comments